openqa-prepare-mm-setup

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env perl
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+use strict;
+use autodie ':all';
+
+system("zypper in -y os-autoinst-openvswitch");
+system("systemctl enable --now os-autoinst-openvswitch");
+
+my $bridge = 1;
+my $openvswitch = "/etc/sysconfig/os-autoinst-openvswitch";
+open my $fh, '>', $openvswitch or die "Failed to open $openvswitch for writing!";
```

Comment:
The `die` isn't useful as autodie is used. The same counts for the other occurrences.